package.json

```diff
@@ -18,17 +18,23 @@
   },
   "scripts": {
     "lint": "eslint packages --cache --fix --ext=jsx --ext=js",
+    "test": "jest",
+    "test:watch": "DEBUG=jest-mongodb:* jest --watch --verbose",
     "bump": "./bump.sh",
     "dev": "./dev.sh"
   },
   "devDependencies": {
+    "@shelf/jest-mongodb": "^1.0.1",
+    "apollo-fetch": "^0.7.0",
```

Comment:
apollo-fetch is deprecated as it seems. Maybe this would also work? https://github.com/prisma/graphql-request

Reply:
Oh wow I didn't see that it is deprecated, but in that case I'd favor just using Apollo Link because we use the Apollo libraries on the server, too, I would create a convenience fetch method with the same signature as apolloFetch and then invoke it that way (apollo docs):

```
import { execute, makePromise } from 'apollo-link';
import { HttpLink } from 'apollo-link-http';
import gql from 'graphql-tag';

const uri = 'http://api.githunt.com/graphql';
const link = new HttpLink({ uri });

const operation = {
  query: gql`query { hello }`,
  variables: {} //optional
  operationName: {} //optional
  context: {} //optional
  extensions: {} //optional
};

// execute returns an Observable so it can be subscribed to
execute(link, operation).subscribe({
  next: data => console.log(`received data: ${JSON.stringify(data, null, 2)}`),
  error: error => console.log(`received error ${error}`),
  complete: () => console.log(`complete`),
})

// For single execution operations, a Promise can be used
makePromise(execute(link, operation))
  .then(data => console.log(`received data ${JSON.stringify(data, null, 2)}`))
  .catch(error => console.log(`received error ${error}`))
```